src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,43 +1,67 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { FullTodo } from '../../types/FullTodo';
+import classNames from 'classnames';
 
-export const TodoModal: React.FC = () => {
-  return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
-
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
-
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
-          </header>
-
-          <div className="modal-card-body">
-            <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
-            </p>
-
-            <p className="block" data-cy="modal-user">
-              {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
-
-              {' by '}
-
-              <a href="mailto:Sincere@april.biz">Leanne Graham</a>
-            </p>
+type Props = {
+  selectedTodo: FullTodo | null;
+  setSelectedTodo: (todo: FullTodo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  setSelectedTodo,
+}) => (
+  <div
+    className={classNames('modal', {
```

Comment:
and here in this component you add the useEffect and the user state that will fetch the user info by selectedTodo.userId
```
 getUser(selectedTodo.userId)
      .then(data => {
        setUser(data);
      })
```